test/community-integration-test-values.yaml

```diff
@@ -2,6 +2,7 @@
 repository:
   resources:
     requests:
+      cpu: "0.01"
```

Comment:
on GHA we only have 2 cpu to run the community tests so I put here the minimum value to make it work and still having a value to be sure that are really handled